DatadogSessionReplay/Tests/DatadogSessionReplayTests/Processor/Flattening/NodesFlattenerTests.swift

```diff
@@ -0,0 +1,118 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import XCTest
+import Datadog
+@testable import DatadogSessionReplay
+@testable import TestUtilities
+
+class NodesFlattenerTests: XCTestCase {
```

Comment:
@ncreated Please take a look whenever you have a free moment. 

I actually think some of those tests have wrong expected values. Maybe we could pair on this a little next week?